buzsaki_lab_to_nwb/huszar_hippocampus_dynamics/behaviorinterface.py

```diff
@@ -122,11 +172,29 @@ def run_conversion(self, nwbfile: NWBFile, metadata: dict, stub_test: bool = Fal
 
         unit = "cm"
         conversion = 100.0  # cm to m TODO: Double check if this is the meaning.
```

Comment:
@garrettmflynn Oh, one last thing I also noticed about the file

I wanted to ask to make sure this is correct in the file/data

We have an example element of the (x,y) `data` of `(70.28238529411765, 44.38839411764705
)`

It has `unit="cm"` and `conversion=100.0`

This means, in NWB, that when I multiply the `data` by the `conversion` I get the correct `unit`

So the true interpretation of the source data is the value `(7_000.28238529411765, 4_400.38839411764705
)` centimeters?

I will also mention we've gotten requests to not use this style of representation when it can be avoided based on data type (i.e., if `float64`, just set `unit="true_unit_of_data"` and `conversion=1.0` to avoid confusion

Reply:
Ah good catch. This is a 79x79cm maze, so these coordinates do not need a conversion factor.

The array values are float64. So it would be good to just set the unit as "true_unit_of_data"?

Reply:
If the true unit of the data is `"cm"`, then yes I would set `conversion=1.0` (or not override it at all since that is default) and set `unit="cm"`